app/models/chargeback.rb

```diff
@@ -170,4 +170,8 @@ def self.get_report_time_range(options, interval, tz)
   def self.report_cb_model(model)
     model.gsub(/^Chargeback/, "")
   end
+
+  def self.chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
> @simon3z : I wanted to enumerate the subclasses rather than match them with start_with? so that when we refactor this for provider plugability we will have an idea on what might actually come here

@martinpovolny I understand and agree on the purpose, I disagree on the implementation.
The base class should not statically know about the subclasses.

Since it's `editor.rb` that has to deal with these classes (and should know about them), I think this list belongs there.

Anyway, for the purpose of extensibility, I'd like to see the future refactoring base on duck typing rather than specific known classes.